infra/scoped/auth0-resource-server.tf

```
@@ -44,3 +44,11 @@ resource "auth0_resource_server" "identity_api" {
   token_lifetime_for_web                          = 60 * 60 // 1 hour
   skip_consent_for_verifiable_first_party_clients = true
 }
+
+resource "auth0_client_grant" "dev_and_test" {
+  for_each = toset(terraform.workspace != "prod" ? local.stage_test_client_ids : [])
```

This seems like a weird way to phrase the ternary.
```suggestion
  for_each = toset(terraform.workspace == "staging" ? local.stage_test_client_ids : [])
```

---

Yeah, I copied this from elsewhere but I'll change both instances